src/DirectWebServices/InvestmentsConnector/InvestmentsConnector.ts

```diff
@@ -113,7 +120,13 @@ export class InvestmentsConnector extends DWSConnector {
 
             converter.parse({ json });
 
-            this.dataTables[type].setColumns(converter.getTable().getColumns());
+            if (this.convertersToUse.find(c => c.key === type)?.children) {
+                for (const table of converter.getTables()) {
+                    this.dataTables[table.id].setColumns(table.getColumns());
```

Comment:
As an addition to this I would also add the metadata from the specific tables: `this.dataTables[table.id].metadata = table.metadata;`